docs/learn/learn-parachains.md

```diff
@@ -16,12 +16,14 @@ on the parachain development guide.
 
 :::
 
+## Definition of a Parachain
+
 A parachain is an application-specific data structure that is globally coherent and validatable by
 the validators of the Relay Chain. They take their name from the concept of parallelized chains that
 run parallel to the Relay Chain. Most commonly, a parachain will take the form of a blockchain, but
 there is no specific need for them to be actual blockchains.
 
-![One parachain](../assets/network/one_parachain.png)
+![One parachain](../assets/one-parachain.png)
```

Comment:
Can the old `one_parachain.png` image be moved into `./assets/_legacy` or removed?

Reply:
moved to _legacy. Will do so for other figures in the future ;)